src/Layout.php

```diff
@@ -6,11 +6,4 @@
 
 class Layout extends View
 {
-    /**
-     * Add a loadable View.
-     */
-    public function addRightPanel(Panel\Loadable $panel): Panel\Loadable
-    {
-        return $this->getOwner()->add($panel, 'RightPanel');
```

Comment:
This was setting `RightPanel` `region` for template. 👍 not needing it anymore, can we remove `RightPanel` region from the templates then?

Reply:
Right! Same with Modals too. I remove both.